src/video_emotion/facial_recognition/t/test_facial_recognition.py

```diff
@@ -1,13 +1,13 @@
 from video_emotion.facial_recognition.facial_recognition import analyze_video
 from utilities.filehandler.handle_path import get_absolute_path
 
+
 def test_facial_recognition_of_at_least_one_face():
-    test_filename = get_absolute_path("video_emotion/facial_recognition/t/te"
-                                      + "st_facial_recognition/Fun_at_a_Fair.mp4")
+    test_filename = (get_absolute_path
+                     ("video_emotion/facial_recognition/t/te" +
+                      "st_facial_recognition/Fun_at_a_Fair.mp4"))
```

Comment:
Would probably (if possible) make the string cut differently so the "test" word isn't cut up.

Reply:
Done